subprojects/packagefiles/lcms2/meson.build

```diff
@@ -0,0 +1,74 @@
+project('Little-CMS', 'c',
+  version: '2.12'
+)
+
+library_version = '2.0.12'
+
+cc = meson.get_compiler('c')
+
+lcms2_srcs = [ 'src/cmsalpha.c'
+        , 'src/cmscgats.c'
```

Comment:
I have just noticed that here in this one place there are commas at the beginning of each line. Which now that I look again, is a general style in the libopenjp2 wrap... madness.

Can this be moved to proper formatting, and include commas after the last element too (as that is the real rationale for making them at the beginning, so that adding new lines doesn't require editing an existing line to add the comma -- but this isn't json. :D)